fv/ingress_egress_test.go

```diff
@@ -49,10 +49,10 @@ var _ = Context("with initialized Felix, etcd datastore, 3 workloads", func() {
 		defaultProfile := api.NewProfile()
 		defaultProfile.Name = "default"
 		defaultProfile.Spec.LabelsToApply = map[string]string{"default": ""}
-		defaultProfile.Spec.EgressRules = []api.Rule{{Action: "allow"}}
+		defaultProfile.Spec.EgressRules = []api.Rule{{Action: api.Allow}}
 		defaultProfile.Spec.IngressRules = []api.Rule{{
-			Action: "allow",
-			Source: api.EntityRule{Tag: "default"},
+			Action: api.Allow,
+			Source: api.EntityRule{Selector: "default == ''"},
```

Comment:
`has(default)` i think is the preferred way of doing this

Reply:
Figured this way you don't match say, `default: whatever` that might be on some other pod for some reason - it strictly matches the profile's label of `default: ""`

Reply:
Either should be fine; it's up to the user really.  Probably good to have a mix in the tests.